packages/containers/psammead-timestamp-container/src/index.stories.jsx

```diff
@@ -8,8 +8,45 @@ import {
   text,
   withKnobs,
 } from '@storybook/addon-knobs';
-import moment from 'moment-timezone';
-import Timestamp from '.';
+import moment from 'moment';
```

Comment:
Same question as comment above.

Reply:
Same as above. I believe we will applying the right timezone in the formatting functions and we don't need in the stories either.